### DISCUSSION TITLE
Update-Installation: Restart = Reboot ?

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [Open-Shell/Open-Shell-Menu#1708] [General]
Open Shell Menu Update Installation: it is not clear wether 
1. to close Explorer and Restart or 
2. no Explorer closing and a "Reboot will be required" 
will really reboot Windows at ONCE or later or what. o_O
Please write exactly what will happen by each option.
Not only I am feared to lose all my work by these rare infos.
thx

**Comment by ge0rdi** [comment]
In case there are some `Open-Shell` components still loaded in Explorer (and cannot be just unloaded) installer displays the prompt to either restart Explorer (process that uses files that have to be removed/replaced) or restart whole machine (which in fact terminates Explorer too and those files can be removed/replaced).

This is pretty standard behavior of Windows installer (not specific to `Open-Shell`).

Restarting Explorer is pretty safe (the same thing as if you do it from Task Manager). All programs will stay running as they were, only Explorer windows will get closed.
